HerPublicWebsite.BusinessLogic/ExternalServices/EmailSending/GovUkNotifyApi.cs

```diff
@@ -94,6 +94,50 @@ string custodianCode
             };
             SendEmail(emailModel);
         }
+
+        public void SendFollowUpEmail
+        (
+            ReferralRequest referralRequest,
+            string followUpLink
+        ) {
+            var template = govUkNotifyConfig.ReferralFollowUpTemplate;
+            LocalAuthorityData.LocalAuthorityDetails localAuthorityDetails;
+            try
+            {
+                localAuthorityDetails = LocalAuthorityData.LocalAuthorityDetailsByCustodianCode[referralRequest.CustodianCode];
+            }
+            catch (KeyNotFoundException ex)
+            {
+                logger.LogError
+                (
+                    ex,
+                    "Attempted to send follow up email with invalid custodian code \"{CustodianCode}\"",
+                    referralRequest.CustodianCode
+                );
+                throw new ArgumentOutOfRangeException
```

Comment:
Yeah good point. I think it would cause issues with other emails being sent. I'll log it and set it as a blank string like you suggest. I think it may look a bit strange in an email. But on consideration, I can't see a situation where this is likely to occur since we're using a referral request generated with an existing custodian code to index the LAs.